source/index.ts

```diff
@@ -608,8 +612,8 @@ export const assert: Assert = {
 	},
 	buffer: (value: unknown): asserts value is Buffer => assertType(is.buffer(value), 'Buffer', value),
 	blob: (value: unknown): asserts value is Blob => assertType(is.blob(value), 'Blob', value),
-	nullOrUndefined: (value: unknown): asserts value is null | undefined => assertType(is.nullOrUndefined(value), AssertionTypeDescription.nullOrUndefined, value), // eslint-disable-line @typescript-eslint/ban-types
-	object: (value: unknown): asserts value is object => assertType(is.object(value), 'Object', value), // eslint-disable-line @typescript-eslint/ban-types
+	nullOrUndefined: /* eslint-disable-line @typescript-eslint/ban-types */ (value: unknown): asserts value is null | undefined => assertType(is.nullOrUndefined(value), AssertionTypeDescription.nullOrUndefined, value),
+	object: (value: unknown): asserts value is Record<string, unknown> => assertType(is.object(value), 'Object', value),
```

Comment:
Don't make unrelated changes. This change is incorrect.

Reply:
<img width="1133" alt="image" src="https://user-images.githubusercontent.com/39019913/173227931-fa99e9ff-ff16-4145-96eb-d358ed1a5da6.png">
Because lint can't pass，I'm so sad.

Reply:
> Don't make unrelated changes. This change is incorrect.

Ok, because I resolved a conflict and lost the lint disable rule, sorry